### ISSUE TITLE
[Tooltip][Accessibility] Tooltip should be dismissable by pressing ESC

### ISSUE THREAD

**Issue by bmeersma** [OfficeDev/office-ui-fabric-react#11720]
Hoverable UI should be dismissable without moving pointer or keyboard focus (i.e. pressing Esc). The tooltip control does not seem follow this requirement.

https://www.w3.org/WAI/WCAG21/Understanding/content-on-hover-or-focus.html

**Comment by xugao** [comment]
@bmeersma  - thanks for the report and i can repro. Will look into fixing

**Comment by xugao** [comment]
@bmeersma - sorry i take that back. I checked https://developer.microsoft.com/en-us/fabric#/controls/web/tooltip and it's dismissed when ESC is pressed. could you elaborate on detailed steps for repro?

**Comment by bmeersma** [comment]
Interesting, I was looking at the internal site and I can repro, but on the external site it's no repro. Maybe it's a bug with the internal site rather than the control?

**Comment by xugao** [comment]
@bmeersma - what do mean by internal site? if you no longer think this is our issue, pls help close :)
